modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -35,35 +35,13 @@ services can cause you to run out of resources.
 Deployments of later versions of {rh-openstack} that have the OVN Octavia driver disabled also
 use the Amphora driver. They are subject to the same resource concerns as earlier versions of {rh-openstack}.
 
-* Octavia {rh-openstack} versions before 13.0.13 do not support UDP listeners. Therefore,
-{product-title} UDP services are not supported.
-
-* Octavia {rh-openstack} versions before 13.0.13 cannot listen to multiple protocols on the
-same port. Services that expose the same port to different protocols, like TCP
-and UDP, are not supported.
-
 * Kuryr SDN does not support automatic unidling by a service.
 
 [discrete]
 [id="openstack-go-limitations_{context}"]
 == {rh-openstack} environment limitations
 
-There are limitations when using Kuryr SDN that depend on your deployment environment.
-
-Because of Octavia's lack of support for the UDP protocol and multiple listeners, if the {rh-openstack} version is earlier than 13.0.13, Kuryr forces pods to use TCP for DNS resolution.
-
-In Go versions 1.12 and earlier, applications that are compiled with CGO support disabled use UDP only. In this case,
-the native Go resolver does not recognize the `use-vc` option in `resolv.conf`, which controls whether TCP is forced for DNS resolution.
-As a result, UDP is still used for DNS resolution, which fails.
-
-To ensure that TCP forcing is allowed, compile applications either with the environment variable `CGO_ENABLED` set to `1`, i.e. `CGO_ENABLED=1`, or ensure that the variable is absent.
-
-In Go versions 1.13 and later, TCP is used automatically if DNS resolution using UDP fails.
-
-[NOTE]
-====
-musl-based containers, including Alpine-based containers, do not support the `use-vc` option.
-====
+* musl-based containers, including Alpine-based containers, do not support the `use-vc` option.
```

Comment:
Seems like this line can be removed too? It's just a workaround for the DNS problem that no longer exists in 16.